.github/workflows/publish.yml

```diff
@@ -0,0 +1,233 @@
+name: CI
```

Comment:
nit: `name: CI (prod)` to distinguish it from name in `ci.yml`? Also, I would use `name: CI (PR)` in ci.yml.